### ISSUE TITLE
Weird Screen Tearing Issue on vWii

### ISSUE THREAD

**Issue by MayorBryce** [GaryOderNichts/evwii#8]
Hey there. So I've got evWii set up and I set up the screen to remove the overscan. However, a rather weird tearing issue started to occur. I have no screenshots myself, but I found this similar Gbatemp issue.

I've attached my config file inside the zip file (for some reason, GitHub doesn't support JSON files).

[gbatemp issue](https://gbatemp.net/threads/nasty-screen-tearing-out-of-the-blue-on-vwii-480p.639931/)

[evWii.zip](https://github.com/GaryOderNichts/evwii/files/12836017/evWii.zip)

**Comment by Ingunar** [comment]
Well, your config does not look corrupted:
```json
{
    "storageitems": {
        "dmcuDRC576i_xEnd": 720,
        "dmcuDRC576i_xSize": 704,
        "dmcuDRC576i_xStart": 8,
        "dmcuDRC576i_yEnd": 576,
        "dmcuDRC576i_ySize": 576,
        "dmcuDRC576i_yStart": 0,
        "dmcuDRC_xEnd": 720,
        "dmcuDRC_xSize": 704,
        "dmcuDRC_xStart": 8,
        "dmcuDRC_yEnd": 480,
        "dmcuDRC_ySize": 480,
        "dmcuDRC_yStart": 0,
        "dmcuTV576i_xEnd": 720,
        "dmcuTV576i_xSize": 704,
        "dmcuTV576i_xStart": 8,
        "dmcuTV576i_yEnd": 576,
        "dmcuTV576i_ySize": 576,
        "dmcuTV576i_yStart": 0,
        "dmcuTV_xEnd": 720,
        "dmcuTV_xSize": 687,
        "dmcuTV_xStart": 16,
        "dmcuTV_yEnd": 480,
        "dmcuTV_ySize": 458,
        "dmcuTV_yStart": 10,
        "enable4sPower": 1
    }
}
```
I couldn't replicate this.
Could you give us any additional information, or record a video?
How does the tearing look like for you?
When does it happen?
Could you provide the link to the GBAtemp issue?

**Comment by MayorBryce** [comment]
I just edited it to show the GBAtemp issue, I had a link there but it wasn't displaying.

https://gbatemp.net/threads/nasty-screen-tearing-out-of-the-blue-on-vwii-480p.639931/

**Comment by Ingunar** [comment]
So it is when 480p was set.
I didn't manage to replicate it BUT, my TV was unable to initialize the signal properly on the first boot after the change to 480p.
(So that was strange, maybe not related)

Are you using 480p mode too?
What region is your WiiU?
Are you using composite cable or other?

**Comment by GaryOderNichts** [comment]
Are you sure this doesn't happen without evWii?

**Comment by MayorBryce** [comment]
I am using 480p mode.
It is a US Wii U.
I am using an HDMI cable that is capable of 1080p. I originally had an mClassic hooked up, but I turned it off when I tested this issue.

My apologies, it seems the issue occurs without evWii. Upon further inspection, it only occurs in injects with and without evWii. At first, I thought it had to do with c2w Patcher, but removing it didn't solve the problem. It also occurs regardless of whether it's installed to the NAND or a Hard Drive.
